src/views/linearProgress.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { Header } from './header';
+import { Button, LinearProgress } from 'react-native-elements';
+
+const LinearProgressAPI: React.FunctionComponent = () => {
+  const [progress, setProgress] = React.useState<number>(0);
+
+  React.useEffect(() => {
+    let subs = true;
+    if (progress < 1 && progress !== 0) {
+      setTimeout(() => {
+        if (subs) {
+          setProgress(progress + 0.1);
```

Comment:
This value change is too small. The state was updating so much, that it was slowing down the animation. `+ 1` worked much better.

Reply:
Value must be between 0 to 1